app/components/primer/image_crop.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Primer
+  # A client-side mechanism to crop images
+  class ImageCrop < Primer::Component
+    # A loading indicator that is shown while the image is loading.
+    renders_one :loading, lambda { |**system_arguments|
+      system_arguments[:tag] = :div
```

Comment:
why always a `div`? Looking at the docs the "loading slot" could be an `img`
The `Cropper with a custom loader` example could look like:

```
    #   <%= render(Primer::ImageCrop.new(src: "https://github.com/koddsson.png", rounded: false)) do |cropper| %>
    #     <% cropper.loading(tag: :img, src: "spinner.gif", style: "width: 120px") %>
    #   <% end %>
```

Reply:
Good shout! I probably meant to make this a logical or operator.

```suggestion
      system_arguments[:tag] ||= :div
```